cpp/models/abm/analyze_result.h

```diff
@@ -0,0 +1,128 @@
+                return model.parameters.template get<IncubationPeriod>()[{VirusVariant::Wildtype, age_group}];
+            });
+            param_percentil(node, [age_group](auto&& model) -> auto& {
+                return model.parameters
+                    .template get<SusceptibleToExposedByInfectedNoSymptoms>()[{VirusVariant::Wildtype, age_group}];
+            });
+            param_percentil(node, [age_group](auto&& model) -> auto& {
+                return model.parameters
+                    .template get<SusceptibleToExposedByInfectedSymptoms>()[{VirusVariant::Wildtype, age_group}];
+            });
+            param_percentil(node, [age_group](auto&& model) -> auto& {
+                return model.parameters
+                    .template get<InfectedNoSymptomsToSymptoms>()[{VirusVariant::Wildtype, age_group}];
+            });
+            param_percentil(node, [age_group](auto&& model) -> auto& {
+                return model.parameters
+                    .template get<InfectedNoSymptomsToRecovered>()[{VirusVariant::Wildtype, age_group}];
+            });
+            param_percentil(node, [age_group](auto&& model) -> auto& {
+                return model.parameters
+                    .template get<InfectedSymptomsToRecovered>()[{VirusVariant::Wildtype, age_group}];
+            });
+            param_percentil(node, [age_group](auto&& model) -> auto& {
+                return model.parameters
+                    .template get<InfectedSymptomsToRecovered>()[{VirusVariant::Wildtype, age_group}];
+            });
+            param_percentil(node, [age_group](auto&& model) -> auto& {
+                return model.parameters.template get<InfectedSymptomsToSevere>()[{VirusVariant::Wildtype, age_group}];
+            });
+            param_percentil(node, [age_group](auto&& model) -> auto& {
+                return model.parameters.template get<SevereToCritical>()[{VirusVariant::Wildtype, age_group}];
+            });
+            param_percentil(node, [age_group](auto&& model) -> auto& {
+                return model.parameters.template get<SevereToRecovered>()[{VirusVariant::Wildtype, age_group}];
+            });
+            param_percentil(node, [age_group](auto&& model) -> auto& {
+                return model.parameters.template get<CriticalToDead>()[{VirusVariant::Wildtype, age_group}];
+            });
+            param_percentil(node, [age_group](auto&& model) -> auto& {
+                return model.parameters.template get<CriticalToRecovered>()[{VirusVariant::Wildtype, age_group}];
+            });
+            param_percentil(node, [age_group](auto&& model) -> auto& {
+                return model.parameters.template get<RecoveredToSusceptible>()[{VirusVariant::Wildtype, age_group}];
+            });
+            param_percentil(node, [age_group](auto&& model) -> auto& {
+                return model.parameters.template get<DetectInfection>()[{VirusVariant::Wildtype, age_group}];
+            });
+        }
```

Comment:
These parameters do not contain an AgeGroup in their type. I don't think it makes sense to separate them into different nodes.

Reply:
I added the rest of the parameters, except for the ones returning non-double value, including:
- InfectionProtectionFactor, SeverityProtectionFactor, HighViralLoadProtectionFactor (return InputFunctionForProtectionLevel)
-  WorkRatio, SchoolRatio, SocialEventRate (return  DampingMatrixExpression<Dampings<Damping<ColumnVectorShape>>>)
- AgeGroupGotoSchool (return set<AgeGroup>)

Reply:
Thanks for adding the parameters. As discussed, we can also have the ViralLoadDistributions and InfectivityDistributions computed by computing the percentiles of the individual parameters and the AntigenTest and PCRTest similarly, although this is mathematically not perfect.
Also, please add a loop for all virus variants for those parameters that depend on them. A loop for MaskProtection with the different MaskTypes would also be nice, if possible.

Reply:
Yes, I added the ViralLoadDistributions and InfectivityDistributions and a loop for all virus variants.
However, the AntigenTest and PCRTest are not in the Parameter set of the World but are created during the design of TestingStrategy. I will leave them out for now.

Reply:
Thanks for the loops. For the distribution type parameters, I'm not sure if they now compute percentiles for all "interior" elements, since these would have to be accessed explicitly, like `get_params(param).viral_load_peak` or something along these lines. Compare with the other conversation.